File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.cpp
class Solution {
public:
    bool uniqueOccurrences(vector<int>& arr)
    {
    
    unordered_map<int,int> m;
    
    
    for(int i=0;i<arr.size();i++){
        m[arr[i]]++;
    }
    vector<int>ans;
   
    for(auto i:m) ans.push_back(i.second);
    
    
    set<int>st(ans.begin(),ans.end());
    
    int n=ans.size();
    int y=st.size();
    
    
    if(n==y) return true;
    return false;

    }
};

File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py
class Solution:
    def uniqueOccurrences(self, arr: List[int]) -> bool:
        c=[]
        a=set(arr)
        for i in a:
            if arr.count(i) in c:
                return False
            else:
                c.append(arr.count(i))
        return True